js/lang/fr.js

```diff
@@ -44,7 +44,7 @@ app.lang['fr'] = (function () {
 		'payment-details.back': 'Retourner',
 		'payment-request.data.invalid': '"data" est invalide',
 		'payment-request.status.invalid': '"status" est invalide',
-		'payment-status.unconfirmed.message': 'Merci!',
+		'payment-status.unconfirmed.message': 'Merci !',
```

Comment:
Is a space needed after the word and before punctuation? Or was this a mistake?

Reply:
In French there's a space before some punctuations (!, ?, and some others). This is needed :)

Reply:
I see. Learning everyday :)